.circleci/config.yml

```diff
@@ -99,11 +99,32 @@ jobs:
           command: |
             mkdir -p /tmp/test-results
             trap "go-junit-report < /tmp/test-results/go-test.out > /tmp/test-results/go-test-report.xml" EXIT
-            go run build/ci.go test -v | tee /tmp/test-results/go-test.out
+            go run build/ci.go test -v -race | tee /tmp/test-results/go-test.out
       - store_artifacts:
           path: /tmp/test-results
           destination: raw-test-output
+      - store_test_results:
+          path: /tmp/test-results
 
+  unit-tests-race:
```

Comment:
I think we could use yaml anchors to avoid duplicating most of the unit test config. See https://discuss.circleci.com/t/using-defaults-syntax-in-config-yaml-aka-yaml-anchors/16168

Reply:
c/p my reply in our private chat here:
For YAML anchor, the CircleCI gives this error: `stat build/ci.go: no such file or directory`, so looks like it cannot separate steps whilst share the attach_workspace , that leaves the usage of this anchor very limited. For our specific case, we can only anchor executor and resource_class, which doesn't seem to be an optimization.

Reply:
tl;dr
we cannot use yaml anchor

Reply:
I figured it out using YAML anchor